internal/manager/task_stash_box_tag.go

```diff
@@ -133,14 +122,8 @@ func (t *StashBoxBatchTagTask) findStashBoxPerformer(ctx context.Context) (*mode
 				}
 			}
 		}
-	} else {
-		var name string
-		if t.name != nil {
-			name = *t.name
-		} else {
-			name = t.performer.Name
-		}
-		performer, err = client.FindPerformerByName(ctx, name)
+	case t.name != nil:
+		performer, err = client.FindPerformerByName(ctx, *t.name)
```

Comment:
I think this reads better if the name and stash id cases are located next to each other.